### PR TITLE
Remove stack build

### PR DESCRIPTION
**Pull request by newhoggy**
As suggested by this comment: https://github.com/input-output-hk/cardano-prelude/pull/121#issuecomment-678937575

### CODE REVIEW

File: scripts/buildkite/default.nix [hunk 1]
```diff
@@ -1,22 +1,40 @@
 { system ? builtins.currentSystem
+, crossSystem ? null
```
Comment (rvl) [inline scripts/buildkite/default.nix:2]: The `scrips/buildkite` directory could be deleted, right?

**Review by rvl — APPROVED** [review]
:+1:

File: scripts/buildkite/default.nix (hunk 1 above)
Comment (newhoggy) [reply scripts/buildkite/default.nix:2]: I left the `cabal check` buildkite script there, but can delete if preferred.

Comment (newhoggy) [reply scripts/buildkite/default.nix:2]: cabal check is removed.

Comment (newhoggy) [reply scripts/buildkite/default.nix:2]: Build failed when I removed it so I put it back.  Maybe remove it in a future PR.

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**